skyvern/forge/agent.py

```diff
@@ -342,21 +344,21 @@ async def execute_step(
                 task_id=task.task_id,
                 step_id=step.step_id,
             )
-            await self.update_step(
-                step=step,
-                status=StepStatus.failed,
-            )
-            task = await self.update_task(
-                task,
-                status=TaskStatus.failed,
-                failure_reason=e.message,
-            )
-            await self.send_task_response(
-                task=task,
-                last_step=step,
-                api_key=api_key,
-                close_browser_on_completion=close_browser_on_completion,
-            )
+            try:
```

Comment:
oops i didn't see your PR until now. I already fixed in this PR: https://github.com/Skyvern-AI/skyvern-cloud/commit/358f04754bda789ebcc280132a95df84a869da7d#diff-97d27efc4f9823539ec219d1cc210dd6a7bf7d4c5ef4b1f9a1dfe937eff5774e